docs/KIBANA_CI.md

```diff
@@ -0,0 +1,54 @@
+# Kibana performance testing on CI
+
+## Overview
+At the moment we got 2 ways to do performance testing: branch testing and cloud testing
+## Branch testing
+Use it to test your feature branch and compare results with master.
+
+Start a new run:
+- go to [load-testing](https://kibana-ci.elastic.co/view/Kibana/job/elastic+kibana+load-testing/) job
+- click 'Build with Parameters'
+- enter your github username & branch name, simulation classes, uncheck 'INGEST RESULTS' if you don't need it in Kibana stats  
+- click 'Build'
+![branch testing job](img/Jenkins_branch_testing.png)
+
+Process:
+- build Kibana from branch source code
+- start Kibana and ES using FTR
```

Comment:
probably you could add how to do it here or link it to the doc which tells how to.

Reply:
Rashmi, I think the `Start a new run:` are the steps how.  The `Process:` part is saying what the job does.  Not what you have to do.

Reply:
Addressed in f3804401f36d776e66cf99285e5b44a83d9877f7
Should be clear now :)

Reply:
more clear now . Thanks Dima